Include/cpython/object.h

```diff
@@ -275,6 +275,7 @@ typedef struct _heaptypeobject {
 
 PyAPI_FUNC(const char *) _PyType_Name(PyTypeObject *);
 PyAPI_FUNC(PyObject *) _PyType_Lookup(PyTypeObject *, PyObject *);
+PyAPI_FUNC(PyObject *) _PyType_Fetch(PyTypeObject *, PyObject *);
```

Comment:
I think the naming convention that's evolved in 3.13 is that function variants that return a strong reference have the `Ref` suffix (like `PyDict_GetItemRef`, `PyList_GetItemRef`, `PyWeakref_GetRef`) so maybe `_PyType_LookupRef`?